.github/workflows/gen-db.yml

```diff
@@ -5,11 +5,12 @@
 name: Generate Database
 
 on:
-  # More details on trigger events: https://docs.github.com/en/actions/reference/events-that-trigger-workflows
+  # More details on trigger events: 
+  #  https://docs.github.com/en/actions/reference/events-that-trigger-workflows
+  pull_request:  # added for testing
```

Comment:
We should not do this on a pull_request, as this action essentially builds the binary file for purposes of the website. That should only be done when we are ready to push a change, not when someone opens a pull request.

Reply:
I am pretty sure I added this here just so it would run on *this* PR. in order to test the script.